mysite/mysite/log.py

```diff
@@ -0,0 +1,16 @@
+"""
+Logging methods
+"""
+
+from django.conf import settings
```

Comment:
pep8 imports warn

Should be:

    import os
    from datetime import datetime

    from django.conf import settings